src/engine/sparqlExpressions/DateExpressions.cpp

```diff
@@ -40,6 +40,22 @@ inline auto extractDay = [](std::optional<DateOrLargeYear> d) {
   return Id::makeFromInt(optionalDay.value());
 };
 
+using LiteralOrIri = ad_utility::triple_component::LiteralOrIri;
+using Literal = ad_utility::triple_component::Literal;
+
+inline auto extractStrTimezone =
+    [](std::optional<DateOrLargeYear> d) -> IdOrLiteralOrIri {
+  if (d.has_value()) {
+    auto optionalTz = d.value().getStrTimezone();
+    if (optionalTz.has_value()) {
+      std::string tz = optionalTz.value();
+      return LiteralOrIri{Literal::literalWithNormalizedContent(
+          asNormalizedStringViewUnsafe(std::move(tz)))};
+    }
```

Comment:
Here we have a completely unnecessary string copy. 
```suggestion
    if (optionalTz.has_value()) {
      std::string tz = optionalTz.value();
      return LiteralOrIri{Literal::literalWithNormalizedContent(
          asNormalizedStringViewUnsafe(optionalTz.value())};
    }
```